src/borg/testsuite/archiver.py

```diff
@@ -736,6 +736,22 @@ def test_extract_with_pattern(self):
             self.cmd("extract", self.repository_location + "::test", "fm:input/file1", "fm:*file33*", "input/file2")
         self.assert_equal(sorted(os.listdir("output/input")), ["file1", "file2", "file333"])
 
+    def test_extract_continue(self):
+        self.cmd('init', self.repository_location)
+        self.create_regular_file('file1', size=1024 * 80)
+        self.create_regular_file('file2', size=1024 * 100)
+        self.cmd('create', self.repository_location + '::test', 'input')
+
+        with changedir('output'):
+            self.cmd('extract', self.repository_location + '::test')
+            with open('input/file1', 'ab') as file:
+                file.truncate(1234)
+            with open('input/file2', 'ab') as file:
+                file.write(b'extra bytes')
+            print(self.cmd('extract', self.repository_location + '::test', '--continue'))
+            assert os.path.getsize('input/file1') == 1024 * 80
+            assert os.path.getsize('input/file2') == 1024 * 100
```

Comment:
even better than just checking length would be checking that content is as original file.